lib/service/system.rb

```diff
@@ -76,7 +76,7 @@ def domain_target
       if root?
         "system"
       else
-        "gui/#{Process.uid}"
+        "user/#{Process.uid}"
```

Comment:
Does this mean background services no longer have access to the GUI display?

Reply:
Actually no, they seem to still have access as long as there is an Aqua session for the same user somewhere. I started a non-Homebrew service with **only** the `Background` session type through SSH for the user I'm currently logged on with (using `bootstrap user/501` explicitly). The underlying program shows an NSAlert and it still pops up in my GUI session. So for single-user systems nothing would change really, they don't even need a dedicated user since that's specifically for multi-user only.